docs/flow/modin/engines/ray/cudf_on_ray/frame/partition_manager.rst

```diff
@@ -0,0 +1,14 @@
+cuDFOnRayFrameManager
+"""""""""""""""""""""
+
+The class is specific implementation of ``RayFrameManager``. It serves as and intermediate level
```

Comment:
```suggestion
The class is specific implementation of ``RayFrameManager``. It serves as intermediate level
```

Reply:
fixed

Reply:
this should be `serves as **an** intermediate level` probably :)

Reply:
Probably, yes :)